modern/src/settings/CalendarPage.js

```diff
@@ -66,6 +66,7 @@ const CalendarPage = () => {
                 label={t('sharedName')}
               />
               <DropzoneArea
+                dropzoneText={t('dropzoneText')}
```

Comment:
What about other unlocalized text related to `DropzoneArea`? There are toasts and some other strings.